evap/results/templates/results_index_evaluation.html

```diff
@@ -61,11 +61,7 @@
     {% endif %}
 
     <div data-col="result"{% if not is_subentry %} data-order="{% if evaluation.is_single_result %}{{ evaluation.single_result_rating_result.average }}{% else %}{{ evaluation.avg_grade|default:7 }}{% endif %}"{% endif %}>
-        {% if evaluation.is_single_result %}
-            {% include 'result_widget.html' with question_result=evaluation.single_result_rating_result single_result=True weight=evaluation.weight weight_sum=evaluation.course.evaluation_weight_sum %}
-        {% else %}
-            {% include 'distribution_widget.html' with distribution=evaluation.distribution average=evaluation.avg_grade weight=evaluation.weight weight_sum=evaluation.course.evaluation_weight_sum %}
-        {% endif %}
+        {% include 'distribution_widget.html' with course_or_evaluation=evaluation %}
```

Comment:
Commenting on an arbitrary line for some naming questions. As the widget templates now have some clearer use-cases, we could also rename those:

- `distribution_widget` → `evaluation_or_course_result_widget` or `result_widget_on_index`
- `result_widget` → `question_result_widget` or `result_widget_on_details`
- `result_widget_tooltip` also may need renaming

But none of the names I currently can think of are a great fit, do you have a better idea?

Reply:
i would go for `evaluation_result_widget` (which would also be used for course results, but having that in the names turns it into some very long name that doesn't really help if you ask me), `question_result_widget` and `result_widget_tooltip`.
is the tooltip specific to one of the two widgets? then the prefix can be added.